src/depot_type.h

```diff
@@ -12,10 +12,10 @@
 
 #include "station_type.h"
 
-typedef StationID DepotID; ///< Type for the unique identifier of depots.
+typedef uint16 DepotID; ///< Type for the unique identifier of depots.
 struct Depot;
 
-static const DepotID INVALID_DEPOT = INVALID_STATION;
+static const DepotID INVALID_DEPOT = (DepotID)INVALID_STATION;
```

Comment:
If depots aren't stations, why is INVALID_DEPOT defined using INVALID_STATION ?

Reply:
Because `Order` uses the same field for different things, and in context of #9959 it makes stuff easier if invalid stations and invalid depots are guaranteed to have the same sentinel value.